lib/gooddata/models/schedule.rb

```diff
@@ -13,7 +13,7 @@
 module GoodData
   class Schedule < Rest::Resource
     attr_reader :dirty, :json
-
+    TRIGGER_EXECUTION_STATUS = 'OK+WARNING'
```

Comment:
TRIGGER_EXECUTION_STATUS --> DEFAULT_TRIGGER_EXECUTION_STATUS

Reply:
The TRIGGER_EXECUTION_STATUS was removed because we will check null before create body.
If the trigger_execution_status is null, It will not set in body

Reply:
With the schedule don't have trigger, It doesn't have triggerExecutionStatus.